### DISCUSSION TITLE
Removing Windows 11's now built in clock on second monitor

### DISCUSSION THREAD

**Discussion by kayotic18** [martinet101/ElevenClock#717] [Q&A] [answered]
Can it be done? I really like having the default Win 11 clock on my main monitor but I adore having ElevenClock on my secondary monitor. It's perfect if I have a game or something up on the monitor as I have it showing on Fullscreen but when I browse internet etc. I have both on top of each other.

**Comment by marticliment** [comment, marked as answer]
ElevenClock shouldn't interfiere with the default clock. If you see any kind of overlapping, try resetting ElevenClock to fix that

**Reply by kayotic18** [reply]
I get ya, but I like the clock on the bottom right. I could change the anchor position but ideally I'd have the built in clock off on the secondary monitor like it was before Microsoft *fixed* it.
![Bottom right of 2nd screen](https://user-images.githubusercontent.com/101054333/160812483-af1f5ae8-53cf-4fed-b468-adef7537e1a8.png)

**Reply by marticliment** [reply]
Yes, the thing is that ElevenClock should be hiding the default clock, but it is not. Can you please send me your log?

**Reply by kayotic18** [reply]
Sorry, I restarted and reloaded ElevenClock. Working as intended after reset. 
Cheers mate
